### PR TITLE
feat(pattern/modal): add overflow modifier

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

File: src/docs/Components/Modals/code.mdx [hunk 1]
```diff
@@ -102,6 +102,12 @@ In cases where it is not necessary, you can use a modal without heading, as foll
 
 <Preview path="withouttitle" />
 
+### Scrolling long content
+
+In some cases, the content paragraph can be bigger than the modal's viewport. A dropped shadow on the top of the footer is used to indicate to the user that this is a scrollable content.
+
+<Preview path="big-content" />
```
Comment (nherwegh) [inline src/docs/Components/Modals/code.mdx:109]: Do we have the right shadow property here? Seems harder than the design file

Comment (mohamedMok) [reply src/docs/Components/Modals/code.mdx:109]: probably not! 
Would you please send me the link to make it consistent

Comment (nherwegh) [reply src/docs/Components/Modals/code.mdx:109]: https://share.goabstract.com/78533af2-8f38-4e31-b3d1-b6dbf88effb3?collectionLayerId=a25c68f5-bf2d-4883-b237-1bbf24e202af&mode=build

**Review by tiloyi — APPROVED** [review]

File: src/docs/Components/Modals/code.mdx (hunk 1 above)
Comment (mohamedMok) [reply src/docs/Components/Modals/code.mdx:110]: thanks @nherwegh,  ready for you to review

**Review by nherwegh — APPROVED** [review]

**Pull request merged**